Add Perplexity provider

stachjankowski opened the pull request:
Basic use of the Perplexity API, with no Internet search results.

stachjankowski committed:
Add Perplexity provider

AleksandrFurmenkovOfficial commented:
Seems good, thank you! To finish the new provider addition, please add an example in README.md as is done for other providers.
And update the description in README.md, line 6 and the description in package.json, line 4, with the list of providers.

stachjankowski committed:
docs: complete Perplexity provider documentation

Add Perplexity example to README and update provider lists in README.md and package.json descriptions

stachjankowski commented:
Thank you for the feedback! I've added the Perplexity example to README.md and updated the provider lists in both README.md and package.json descriptions as requested.
